.jenkins/pipelines/coverage_converter.groovy

```diff
@@ -0,0 +1,76 @@
+pipeline {
+    agent {
+        docker {
+            image 'testmnrp/dart-converter:2'
+            args '-u root:root'
+        }
+    }
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+    environment {
+        METRICS_PROJECT_ID = "jenkins_coverage_converter"
+    }
+    stages {
+        stage('Git clone') {
+            steps {
+                git branch: 'master', url: 'https://github.com/platform-platform/monorepo.git'
+            }
+        }
+        stage('Get Dependencies') {
+            steps {
+                dir('metrics/ci_integrations'){
+                    sh 'pub get'
+                }
+                script{
+                    isTested = false
+                }
+            }
+        }
+        stage('Parallel Stage') {
+                    parallel {
```

Comment:
Should we add a [failFast true](https://www.jenkins.io/doc/book/pipeline/syntax/#parallel) flag here to fail the parallel build once the `dartanalyzer ` or `test_coverage` fails? Please, apply this suggestion to all pipelines that contains `parallel` step.
 ```suggestion
        stage('Parallel Stage') {
                    failFast true
                    parallel {
```